### PR TITLE
WFLY-16709 Standardize the maven module descriptions.

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
Issue: https://issues.redhat.com/browse/WFLY-16709

### CODE REVIEW

**Comment by RanabirChakraborty** [comment]
@bstansberry thank you so much for the help. Can you please review this once.

**Review by rhusar — APPROVED** [review]
Thx for doing this!

File: testsuite/scripts/pom.xml [hunk 1]
```diff
@@ -31,6 +31,8 @@
 
     <artifactId>wildfly-testsuite-scripts</artifactId>
 
+    <name>WildFly: Testsuite Scripts</name>
```
Comment (bstansberry) [inline testsuite/scripts/pom.xml:34]: Please make this "WildFly Test Suite: Scripts"

**Review by bstansberry — CHANGES_REQUESTED** [review]
Hi @RanabirChakraborty 

Thanks! There are still a few that need names:

wildfly-ee-9-source-transform-parent
wildfly-feature-pack-parent
wildfly-feature-pack-galleon-content
wildfly-ee-9-parent (which should be "WildFly Preview: Parent")
wildfly-servlet-feature-pack-parent

File: testsuite/scripts/pom.xml (hunk 1 above)
Comment (RanabirChakraborty) [reply testsuite/scripts/pom.xml]: Thanks, @bstansberry I have changed accordingly.

**Review by bstansberry — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks @RanabirChakraborty!

**Pull request merged**